File: codingtest/지폐 접기.cpp
#include<iostream>
#include <vector>
#include<cmath>
#include<algorithm>
using namespace std;
//lv1. ÁöÆó Á¢±â

int solution(vector<int> wallet, vector<int>bill) {
	int answer = 0;
	int walletmax;
	int walletmin;
	int billmax;
	int billmin;
	walletmax = max(wallet[0], wallet[1]);
	walletmin = min(wallet[0], wallet[1]);
	billmax = max(bill[0], bill[1]);
	billmin = min(bill[0], bill[1]);
	while (walletmax < billmax || walletmin < billmin) {
		billmax /= 2;
		answer += 1;

		if (billmax < billmin) {
			swap(billmax, billmin);
		}

	}



	return answer;
}

File: codingtest/pyrhon lv1~/지폐 접기.py
def solution(wallet, bill):
    answer = 0
    while not ((wallet[0] >= bill[0] and wallet[1] >=bill[1]) or (wallet[1] >= bill[0] and wallet[0] >=bill[1])):
        answer +=1
        if(bill[0]>=bill[1]):
            bill[0]=bill[0]//2
        else : 
            bill[1]=bill[1]//2




    return answer